.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "qsharp-language"]
+	path = qsharp-language
+	url = https://github.com/microsoft/qsharp-language.git
+	branch = samarsha/unified-grammar
```

Comment:
this should to `main`, no?

Reply:
The submodule is also not pointing to `main`, so when initialized and checked out it will point to the exact commit listed in the repo. Probably not the intention.